_projects/optimize_caching.md

```diff
@@ -62,23 +69,24 @@ increase performance even further.
 
 ##### CODING PHASE 1
 
-* The Proposed change to not explore unchanged directories has been  
```

Comment:
why is this removed?  Is the `Directory` class related to this?  if so, it would be better to add `Directory` to this first bullet point, revising it a bit, so it is obvious you are clarifying/revising rather than removing a deliverable.

Reply:
Done. Made Directory the first bullet point and added extra explaination.